gr-digital/CMakeLists.txt

```diff
@@ -30,6 +30,11 @@ SET(GR_PKG_DIGITAL_EXAMPLES_DIR ${GR_PKG_DATA_DIR}/examples/digital)
 ########################################################################
 if(ENABLE_GR_DIGITAL)
 
+# Set HAVE_BUILTIN_PARITYL for use in scrambler if compiler is GNU
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+    add_definitions(-DHAVE_BUILTIN_PARITYL)
```

Comment:
Is this true for all gcc/clang versions down to our minimum?

Reply:
I don't know this, but __builtin_parity seem to have been around forever.

Reply:
Hm yeah, definitely much longer than 8.3.0.